### DISCUSSION TITLE
Add some additional languages

### DISCUSSION THREAD

**Discussion by aolko** [pieces-app/support#46] [Ideas]
### Pascal-like
- Pascal
- Object pascal/Delphi
- [Oxygene](https://www.remobjects.com/elements/Oxygene/language.aspx)
### .net and older
- VBA
- VB6
- VB.NET
- [Mercury](https://www.remobjects.com/elements/mercury)

**Comment by caleb-at-pieces** [comment]
Hi there @aolko looks like we added support for VBA in the last pieces os release, I will make a note of these others so they should be available once we release pieces os 9.0.0

**Reply by idiston** [reply]
Unable to find VBA as a language in version 2.10.1 so far
